service/app/controllers/healthmanager/HealthController.java

```diff
@@ -102,9 +102,9 @@
             PropertiesCache.getInstance().getProperty(JsonKey.EKSTEP_AUTHORIZATION));
       }
       headers.put("Content-Type", "application/json");
-      String ekStepBaseUrl = System.getenv(JsonKey.EKSTEP_BASE_URL);
+      String ekStepBaseUrl = System.getenv(JsonKey.SEARCH_SERVICE_API_BASE_URL);
```

Comment:
ProjectUtil has method to get config value directly from either env or properties file.